### PR TITLE
Ls removed location engine.activate() from location examples

### PR DESCRIPTION
**Pull request by langsmith**
Fixes https://github.com/mapbox/mapbox-android-demo/issues/248

### CODE REVIEW

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java [hunk 1]
```diff
@@ -38,7 +38,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // Get the location engine object for later use.
     LocationEngine locationEngine = LocationSource.getLocationEngine(this);
-    locationEngine.activate();
     locationEngine.requestLocationUpdates();
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java]: this line can also be removed

Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java]: For some reason, the example breaks when requestLocationUpdates() is removed. That's why I left it in there.

Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java]: Oh, actually I'll try it now that https://github.com/mapbox/mapbox-android-demo/pull/321#pullrequestreview-36279345 has landed

Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java]: Yea, works after merging master back in. 

`activate();` and  `requestLocationUpdates();` have both been removed

**Pull request merged**